Python/Product/Django/Intellisense/DjangoCompletionSourceBase.cs

```diff
@@ -94,7 +94,7 @@ internal abstract class DjangoCompletionSourceBase : ICompletionSource {
                     "tag"));
             return new FuzzyCompletionSet(
                 "PythonDjangoTags",
-                "Django Tags",
+                Resources.DjangoTagsCompletionSetDisplayName,
```

Comment:
You need two completion sets in the same completion popup. We don't do this because it's ugly and difficult to use.